integration-tests/deployment/ccip/test_helpers.go

```diff
@@ -560,3 +564,66 @@ func deploySingleFeed(
 	return mockTokenFeed.Address, desc, nil
 
 }
+
+// remove duplicates from a slice
+func removeDuplicates[T comparable](slice []T) []T {
+	// Create a map to store unique elements
+	seen := make(map[T]bool)
+	result := []T{}
+
+	// Loop through the slice, adding elements to the map if they haven't been seen before
+	for _, val := range slice {
+		if _, ok := seen[val]; !ok {
+			seen[val] = true
+			result = append(result, val)
+		}
+	}
+	return result
+}
```

Comment:
Out of curiosity I looked up whether such a function was available in the golang [`slices`](https://pkg.go.dev/slices) package and unfortunately didn't find something out of the box that removes all duplicates, but an interesting combo would be to do a sort and then a compact (this can also be done at the call site and therefore can remove this function):

```golang
noDups := slices.Compact(slices.Sorted(slices.Values(slice)))
```

Not as pretty as the functional languages but has the advantage of being standard code and less stuff to maintain.

Reply:
another alternative I was considering was to use a Set initially to inherently avoid duplicates. 

Golang doesn't have native set support, but I've seen us using `mapset "github.com/deckarep/golang-set/v2"` in other places. Any issues with me using that instead?

Reply:
Yah `mapset` sounds fine, we use that in a lot of places already. If its available in the current go.mod I don't see a problem w/ using it.